src/react-components/libraryDetailsView.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect } from 'react';
+import { useAppSelector, useAppDispatch } from '../redux/store';
+import { fetchBooks } from '../redux/libraryDetails';
+import { Link } from 'react-router-dom';
+
+export const LibraryDetailsView = () => {
+  const libraryDetails = useAppSelector(state => state.libraryDetails);
+  const libraryIndex = useAppSelector(state => state.libraryIndex);
+  const dispatch = useAppDispatch();
+
+  const getLibrary = (id: number) => {
```

Comment:
Lines 11-15 make total sense for development. When we incorporate this page's Route into `App.tsx` these could move into that Route element. And then we could use `match.params` within that Route to get the dynamic library id.

Reply:
Yes, sounds good